src/main/scala/no/ndla/listingapi/model/api/Cover.scala

```diff
@@ -28,8 +28,30 @@ case class Label(@(ApiModelProperty@field)(description = "The type of this label
                  @(ApiModelProperty@field)(description = "The label") labels: Seq[String]
                 )
 
+@ApiModel(description = "Meta information for a new cover")
```

Comment:
Nå ligger `Cover` i en pakke `....model.api` så da føler jeg `ApiModels` som navn på filen være litt overkill.
Hva om jeg fjerner `domain` og `api` pakkene, og heller har to filer i `model` pakken; en `Api.scala` og en `Domain.scala`. `Error` filene flyter egentlig litt over i hverandre, så de kunne kanskje blitt slått sammen til én fil.
Strukturen vil da bli slik:
```
model/
    Api.scala
    Domain.scala
    Errors.scala
    search/
        Language.scala
        ReindexResult.scala
        ...
```